ci/ansible/roles/pulp-backup/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+# rhel6 pulp 2.11.z backup
+- name: Stop services before backup
+  service: "name={{ item }} state=stopped"
+  with_items:
+    - httpd
+    - pulp_celerybeat
+    - pulp_resource_manager
+    - pulp_streamer
+    - pulp_workers
+    - qpidd
+  notify: Restart services
+
+- block:
+    - name: Create backups folder
+      file: path={{ remote_bkup_dir }} state=directory
+
+    - name: Create local backups folder
+      local_action: file path={{ local_bkup_dir }} state=directory mode=0777
```

Comment:
The file mode set on this directory created on the control node is hacky and I'd like a better alternative.

It may be better to just trash this step of creating the folder on the control node and require that the caller provide it. The only requirement it must fulfil is existence and permission to write to it.